### DISCUSSION TITLE
RSC-FAULT

### DISCUSSION THREAD

**Discussion by MartinJSa** [cmcgerty/Sofar2mqtt#87] [General]
Hi, Hi, I've woken this morning to find my Sofar2MQTT device in an error state "RSC FAULT ERROR" 
Has anyone seen this before please? A simple power reset doesn't clear it, and the ME3000SP seems to be functioning normally?

![image](https://github.com/user-attachments/assets/eee81fe5-2e91-4554-a60d-ca15d5e9d1df)

**Comment by cliffdude** [comment]
Try to restart the inverter, it is rare but sometimes the RS485 just stops
working. You will have to cut the grid and the solar for the inverter to
shut down.

**Reply by MartinJSa** [reply]
Thanks very much, did a complete shutdown and restart of the inverter and we're back working again. Sorry shouldn't have done that before raising a ticket, but my expectations would have been a CRC Error if the RS485 side had stopped working. Hopefully this will be useful for others in the future.

**Comment by jacoscar** [comment]
So what's the solution to this if we are not home to restart the inverter?

**Comment by double1968** [comment]
Once every 4-5 months, my sofra2mqtt stops communicating with the inverter, but it still remains pingable. The RS485 icon on the inverter display disappears. I've discovered that when communication is interrupted, if I wait a few hours, communication usually resumes overnight. After years of using sofar2mqtt, I think the problem is with the inverter software and not sofar2mqtt.
Even resetting the ESP doesn't allow communication.
My inverter is a ZCS Azzurro, Sfar rebrands
